DataFormats/FTLDigi/src/classes_def.xml

```diff
@@ -4,24 +4,20 @@
    <class name="FTLDataFrame"/>
    <class name="std::vector<FTLDataFrame>"/>
    <class name="FTLDigiCollection"/>
-   <class name="edm::Wrapper<FTLDigiCollection>" />
```

Comment:
Did you verify that only the `Wrapper<T>` is not used? (I don't disagree, on a quick look the entire `FTLDigiCollection` seem unused, in which case I'd proceed to remove them all, but I'd also prefer such cleanup to be done in a separate PR)

Reply:
@makortel sorry if I did something wrong: I just cleaned the entries that were causing the build to fail. I am running a local build now to check if there are other unused entries.

Reply:
The error from `rootcling` is indeed a bit strange. But removing the `Wrapper<T>` from the `classes_def.xml` is unlikely to be the right fix (`rootcling` doesn't know if the `Wrapper<T>` is really needed or not).

Just guessing, would the error disappear if the `DataFormats/Common/interface/Ref.h` would be included in the `classes.h`?

Reply:
Yes, adding that does work for local build.

Reply:
Thanks, interesting discovery. I wonder how including `Ref.h` plays a role here.